packages/adapter-cloudflare/index.js

```diff
@@ -113,6 +113,13 @@ export default function (options = {}) {
 					ASSETS: assets_binding
 				}
 			});
+			if (builder.hasServerTracingFile()) {
+				builder.trace({
```

Comment:
Is there some benefit in testing that the server instrumentation file intercepts those imports correctly in the Cloudflare and Netlify test suite?

Reply:
Added tests